src/inc/corinfo.h

```diff
@@ -2305,6 +2305,11 @@ class ICorStaticInfo
             CORINFO_MODULE_HANDLE   scope
             ) = 0;
 
+    virtual BOOL isInSameVersionBubble(
```

Comment:
The concept of the version bubble should not be explosed on the JIT/EE interface. I do not understand why you have added this - I do not see any call from JIT.

Reply:
Removed. I was trying to test for version bubble from zapinfo.cpp. I didn't realize the implementation of the `IsInSameVersionBubble` was already compiled into crossgen